pkgs/by-name/oc/oci-cli/package.nix

```diff
@@ -26,7 +26,7 @@ in
 py.pkgs.buildPythonApplication rec {
   pname = "oci-cli";
   version = "3.61.0";
-  format = "setuptools";
+  format = "pyproject";
```

Comment:
We should go with `pyproject = true` as mentioned in the [docs](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md).